### PR TITLE
Realex missing notification methods

### PR DESCRIPTION
**Pull request by davidjrice**

### CODE REVIEW

**Comment by andrewpaliga** [comment]
@j-mutter @girasquid Shopify throws exceptions when these methods are missing :(

Code look good?

**Comment by girasquid** [comment]
Code looks good.

File: lib/offsite_payments/integrations/realex.rb [hunk 1]
```diff
@@ -179,6 +180,18 @@ def acknowledge(authcode = nil)
...
 
+        def item_id
+          order_id
+        end
+
+        def transaction_id
+          pasref
+        end
+
+        def test?
```
Comment (j-mutter) [inline lib/offsite_payments/integrations/realex.rb]: Not tested below?

Comment (davidjrice) [inline lib/offsite_payments/integrations/realex.rb]: done!

**Pull request merged**